.github/workflows/e2e-sock-trigger.yaml

```diff
@@ -0,0 +1,67 @@
+name: E2ESoakTrigger
+on:
+  schedule:
+    - cron: '0 */3 * * *'
+  workflow_run:
+    workflows: [ApprovalComment]
+    types: [completed]
+  workflow_dispatch:
+    inputs:
+      git_ref:
+        type: string
+      region:
+        required: true
+        default: 'us-east-1'
+        type: choice
+        options:
+          - "us-east-1"
+          - "us-west-2"
+      cleanup:
+        required: true
+        default: true
+        type: boolean
+permissions:
+    id-token: write # aws-actions/configure-aws-credentials@v4.0.1
+    statuses: write # ./.github/actions/commit-status/start
+jobs:
+  reslove_cluster:
+    runs-on: ubuntu-latest
+    outputs:
+      CREATE_CLUSTER: ${{ steps.create_cluster.outputs.CREATE_CLUSTER }}
+      PREEXISTING: ${{ steps.create_cluster.outputs.PREEXISTING }}
+    steps:
+    - uses: actions/checkout@v4
+      with:
+        ref: ${{ inputs.git_ref }}
+    - name: configure aws credentials
+      uses: aws-actions/configure-aws-credentials@v4.0.1
+      with:
+       role-to-assume: arn:aws:iam::${{ vars.ACCOUNT_ID }}:role/${{ vars.ROLE_NAME }}
+       aws-region: ${{ inputs.region || 'us-east-1' }}
+       role-duration-seconds: 21600
+    - uses: ./.github/actions/e2e/install-eksctl
+      with:
+        version: v0.163.0
+    - id: create_cluster
+      run: |
+        export PREEXISTING=$(eksctl get cluster -o json | jq '.[].Name' | grep soak)
```

Comment:
this assumes that there's only one cluster that has soak in its name. Is there some way to persist values of these clusters so that we can refer to them in the future? Maybe we can create clusters with name of the date they were created in, so that our soak test knows for sure what the name of the cluster should be?